README.md

```diff
@@ -58,15 +58,29 @@ dependencies:
 # Example Playbook
 
 ```yaml
+
 - hosts: localhost
+  vars_files:
+    - ansible_vault_key.yml
   roles:
+    - reallyenglish.apt-repo
```

Comment:
IIRC, if you add dependencies to meta, these can be omitted because `ansible-role-java` depends on `reallyenglish.apt-repo`. a drawback is, the depended roles are always executed before the depending role no matter how. the point is, when some roles are always required for successful play, the role should go to meta. if they are optional, and required for the purpose of demonstration in an example, they should go to `roles` in the playbook.

which is the case here?